taxi/views.py

```diff
@@ -61,6 +61,14 @@ class CarListView(LoginRequiredMixin, generic.ListView):
 class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
+    def post(self, request, *args, **kwargs):
+        car = self.get_object()
+        if request.user in car.drivers.all():
+            car.drivers.remove(request.user)
+        else:
+            car.drivers.add(request.user)
+        return redirect("taxi:car-detail", pk=car.id)
```

Comment:
The `post` method in `CarDetailView` directly modifies the `drivers` many-to-many relationship based on the current user. Ensure that this logic aligns with your application's requirements, as it allows any authenticated user to add or remove themselves from a car's drivers list. Consider adding permission checks if needed.